packages/checkup-plugin-javascript/__tests__/eslint-disable-task-test.ts

```diff
@@ -86,4 +92,28 @@ describe('eslint-disable-task', () => {
       }
     `);
   });
+
+  it('returns action item if there are more than 2 instances of eslint-disable', async () => {
+    const result = await new EslintDisableTask(
+      pluginName,
+      getTaskContext({
+        paths: project.filePaths,
+        config: {
+          tasks: {
+            'javascript/eslint-disables': ['on', ['numEslintDisables']],
```

Comment:
I think we discussed updating the `ActionConfig` so that it wasn't just a string to implicitly mark this as being 'on'.

I think we should have an here, with an actions property. That property can be an object of action configs, that we then read. 

```suggestion
            'javascript/eslint-disables': ['on', {
              'actions': {
                'numEslintDisables': 'on',
                'foo': {
                  'threshold': 10
                }
              }
            }],
```